futaba/sql/models/filter.py

```diff
@@ -265,6 +265,18 @@ def get_filter_immune_users(self, guild):
         self.immune_users_cache[guild].update(user_id for user_id, in result.fetchall())
         return self.immune_users_cache[guild]
 
+    def get_filter_immune_users(self, guild):
+        logger.info(
+            "Getting users about filter immunity in guild '%s' (%d)",
```

Comment:
`'%s'` → `%r`

Reply:
`guild.name` is a string
Also I'm not sure if escaping the name makes a lot of sense

Reply:
`%r` works on everything, including strs
it wouldn't escape anything unless there was a both a `'` and a `"` in the guild name (or something else that needed escaping)